### ISSUE TITLE
CLB Packs

### ISSUE THREAD

**Issue by Timespiraled** [dr4fters/dr4ft#1620]
CLB packs seem to have reverted to 15-card packs rather than 20-card packs after the latest batch of cards were added.

**Comment by tooomm** [comment]
When I start a CLB draft, it shows me 15 cards?

**Comment by Timespiraled** [comment]
> When I start a CLB draft, it shows me 15 cards?

Right, exactly. That's not the correct pack size for the set. It should be like CMB with 20 card packs and 2 guaranteed legendaries (or backgrounds) of any rarity per pack. You can compare to sample packs from draftsim: https://draftsim.com/draft.php?mode=Draft_CLB

**Comment by Timespiraled** [comment]
I have a memory of it doing so, and I know for a fact it used to have correct collation (20 card packs including 2 legends per pack) for CMB as we did months of playtesting using this site for that set.

**Comment by taw** [comment]
I don't have CLB yet as mtgjson does not have ful spoiler yet.

**Comment by tooomm** [comment]
You have CMB1 and CMB2 if I see it correctly. @Timespiraled is reporting that they should have 20 cards booster as well (like CLB).

**Comment by MQGHugs** [comment]
I was doing some digging into this as well, as it seems to generate with 15 cards currently as well, and I'm curious where the CLB set is currently coming from. I didn't see it in the repo you linked yet. It is very possible I just didn't parse the other repos json correctly and didn't see this new set.

**Comment by taw** [comment]
CLB is on mtgjson, so if someone can find out exact rules for the packs, that would be great.

**Comment by MQGHugs** [comment]
The stated rules per pack are a little complex, but it looks like this:
1 Legendary creature or planeswalker
1 Legendary Background
1 Non-legendary rare or mythic rare card
3 Uncommons
13 Commons
Up to 1 foil card of any rarity (Replaces 1 common card. Foil not guaranteed.)
Unsure if they did any sort of messing with the rarity of their legendary backgrounds and legendary creatures.
As for what rules are currently being used by the current pack generator, I'm unsure.
